src/addon-manager-proxy.ts

```diff
@@ -441,16 +460,16 @@ class AddonManagerProxy extends EventEmitter {
             // expecting a reply, so we report the error
             // and just send whatever the current value is.
             console.error('AddonManagerProxy: Failed to setProperty',
-                          propertyName, 'to', propertyValue,
-                          'for device:', deviceId);
+              propertyName, 'to', propertyValue,
```

Comment:
nit: spacing change

Reply:
`typescript-eslint` should take care of them.

Reply:
It seems to work but I need to solve a lot of violations :grimacing: